### Instructions

1. It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

* Set up the hypothesis test.
* Write down all the steps followed for setting up the test.
* Calculate the test statistic by hand and also code it in Python. It should be 4.76190. What decision can you make based on this calculated value?

Hint: here is the statistic table

#### Optional
2. In a packing plant, a machine packs cartons with jars. It is supposed that a new machine will pack faster on the average than the machine currently used. To test that hypothesis, the times it takes each machine to pack ten cartons are recorded. The results, in seconds, are shown in the tables in the file Data/machine.txt. Assume that there is sufficient evidence to conduct the t test, does the data provide sufficient evidence to show if one machine is better than the other.


Hint: use two sample t-test

t= (sample_mean(x1) - sample_mean(x2)) / sqrt(square(s1)/n1 + square(s2)/n2 )

### Problem statement
It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. However, In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg

Is the mean of our sample different than 120 mm Hg due to chance, or because the people who took part in the study actually have a mean systolic blood pressure different from 120 mm Hg?

### Process
Significance Level (α): 0.05 (5%)

### Hypothesis
H0: μ = 120 mm Hg

H1: μ != 120 mm Hg


#### Which distribution should we use?
Our sample size is bigger than 30 but we do not know the poulation standard deviation therefore our sample distribution will follow a a t-student distribution.



In [11]:
import numpy as np
import pandas as pd
import scipy.stats as stats
import matplotlib.pyplot as plt
import seaborn as sns


%matplotlib inline

#### Summary
* population mean μ = 120 mm Hg
* sample mean = 130.1 mm Hg
* sample standard deviation = 21.21 mm Hg
* sample size n = 100

In [18]:
pop_mean = 120
samp_mean = 130.1
samp_std = 21.21
samp_size = 100
signif_level = 0.05
dof = samp_size - 1


99

In [13]:
# Let's compute the standard error:
sem = samp_std/np.sqrt(samp_size)
sem

2.121

#### Z critical

In [19]:
zc = stats.t.ppf(1-0.025, dof)

zc

1.9842169515086827

#### T stat


In [20]:
# Calculation of t-valu

t_stat = (samp_mean - pop_mean) / sem
t_stat

4.761904761904759

#### Conclusion
The t stat value was found to be 4.76 which is higher than our z critical value
Therefore we cannot reject the null hypothesis that H0: μ = 120 mm Hg

t_stat = 4.76 mean we are 4.76 standard deviations away from the mean

In [25]:
p_value = stats.t.sf(t_stat, dof)
p_value


#compare value

3.2813509086043083e-06